subprojects/docs/src/docs/release/notes.md

```diff
@@ -46,9 +46,17 @@ Add-->
 ### Locking of dynamic dependencies
 
 Gradle now provides a mechanism for locking dynamic versions.
-It enables builds to become reproducible even when declaring dependencies using version ranges.
+It makes builds reproducible even when declaring dependencies using version ranges.
 
-You first enable locking on configurations:
+This enables, amongst others, the following scenarios:
+
+* Companies dealing with multi repositories no longer need to rely on `-SNAPSHOT` or changing dependencies,
```

Comment:
Didn't you mean "for" instead of "or" before "changing dependencies"?

Reply:
No, while `SNAPSHOT` are implicitly changing, you could have a company using its own scheme, non maven, for having changing dependencies.